packages/@okta/vuepress-site/use_cases/event_hooks/index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Event Hooks
+excerpt: Use Okta events to drive custom process flows.
+---
+
+# Event Hooks
+
+<ApiLifecycle access="ea" />
+
+## What Are Okta Event Hooks?
+
+Event hooks are outbound calls from Okta, sent when specified events occur in your org. They take the form of HTTPS REST calls to a URL you specify, encapsulating information about the events in JSON objects in the request body. These calls from Okta are meant to be used as triggers for process flows within your own software systems.
+
+To handle event hook calls from Okta, you need to implement a web service with an Internet-accessible endpoint. It's your responsibility to develop the code and to arrange its hosting on a system external to Okta. Okta defines the REST API contract for the requests that it will send.
+
+Event hooks are related to, but different from, Okta [inline hooks](/use_cases/inline_hooks/). Event hooks are meant to provide information about events that occurred, not offer a way to affect the underlying Okta process flow. For that, you need to use inline hooks. Also, unlike inline hooks, event hooks are asynchronous calls, meaning that the process flow that triggered the event hook continues without stopping or waiting for any response from your external service.
+
+You can create a maximum of ten event hooks in your org. Each event hook can be configured to deliver multiple event types.
```

Comment:
Please mention that the limit is 10 _Active_ event hooks.  An org can have, for example, 10 active hooks and 100 inactive hooks.

Reply:
^^ +1. Something like - `You may have at most 10 active and verified event hooks in your org at any point`.

Reply:
Changed to: "You can have a maximum of 10 active and verified event hooks in your org at any point."